__mocks__/globby.js

```diff
@@ -19,7 +19,12 @@ async function globby(path, _options) {
 	}
 
 	// Check package results
-	return resultsPackage[path];
+	if (path.includes('packages')) {
+		if (path.includes('error')) {
+			throw new Error('globby error');
+		}
+		return resultsPackage[path];
+	}
```

Comment:
What should happen when the path doesn't include `packages` or `toolkits`? 
Can this `return Promise.resolve([])` or something otherwise?

Reply:
It's a mock so I know from my tests that it will always contain one of those 😆 But, yeah, can just return an empty array. Can't I just `return [];` as it's an `async` function??

Reply:
Yup, didn't notice the `__mocks__` path